tests/test_semver.py

```diff
@@ -80,3 +80,18 @@ def test_should_versioninfo_isvalid():
 def test_versioninfo_compare_should_raise_when_passed_invalid_value():
     with pytest.raises(TypeError):
         Version(1, 2, 3).compare(4)
+
+
+@pytest.mark.parametrize(
+    "caller, callee",
+    [
+        ((1, 2, 3), (1, 2, 3)),
+        ((1, 2, 3), (1, 2, 4)),
+        ((1, 2, 3), (1, 2, 4)),
```

Comment:
❗ you've got the same test here twice, did you mean 
```suggestion
        ((1, 2, 4), (1, 2, 3)),
```